nucypher/network/nodes.py

```diff
@@ -1363,7 +1363,7 @@ def node_details(node):
                    "fleet_state": node.fleet_state_checksum or 'unknown',
                    "fleet_state_icon": fleet_icon,
                    "domain": node.serving_domain,
-                   'version': nucypher.__version__}
+                   }
```

Comment:
Would we ever want to include the version if `node == self`? Or is it weird to sometimes have version and other times not.

Reply:
Maybe I should just leave this commit out and have the PR tightly-scoped?

Reply:
👌 I could see that - perhaps more discussion is warranted.